runtime/src/main/java/uk/ac/manchester/tornado/runtime/profiler/TimeProfiler.java

```diff
@@ -32,6 +32,8 @@
 
 public class TimeProfiler implements TornadoProfiler {
 
+    public static String NO_TASK_NAME = "noTask";
```

Comment:
Add Java-doc why this taskName is neded. Also, you can relate to the open issue we have to discuss this.